packages/utils/src/snark-artifacts/snark-artifacts.browser.ts

```diff
@@ -1,31 +1,21 @@
-import { GetSnarkArtifactUrls } from "./config"
-import { Proof, SnarkArtifacts, Version } from "../types"
+import { BigNumber, SnarkArtifacts, Version } from "../types"
 
-function MaybeGetSnarkArtifacts(proof: Proof.EDDSA, version?: Version): () => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(
-    proof: Proof.POSEIDON,
-    version?: Version
-): (numberOfInputs: number) => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(
-    proof: Proof.SEMAPHORE,
-    version?: Version
-): (treeDepth: number) => Promise<SnarkArtifacts>
-function MaybeGetSnarkArtifacts(proof: Proof, version?: Version) {
-    switch (proof) {
-        case Proof.POSEIDON:
-            return async (numberOfInputs: number) => GetSnarkArtifactUrls({ proof, numberOfInputs, version })
+export default async function maybeGetSnarkArtifacts(
+    projectName: string,
```

Comment:
I would prefer an enum here. Eg we can use [`Proof`](https://github.com/privacy-scaling-explorations/zk-kit/blob/df57ac2f9c719fd3bc2573fc3ff612b3a634ef27/packages/utils/src/types/index.ts#L26)